gfx/MShockXotto+/pngs_tiles_32x32/character/overlay/wielded/Melee/melee.json

```diff
@@ -0,0 +1,5 @@
+{"id": "overlay_wielded_staff_pipe", "fg": ["overlay_wielded_staff_pipe"], "rotates": true, "bg": []},
+{"id": "overlay_wielded_glaive", "fg": ["overlay_wielded_glaive"], "rotates": true, "bg": []},
```

Comment:
if id and fg are the same then you don't need the json file, unless you're also using the sprite for another id somewhere else